apps/docs/content/guides/platform/compute-and-disk.mdx

```diff
@@ -10,11 +10,9 @@ Every project on the Supabase Platform comes with its own dedicated Postgres ins
 
 The following table describes the base instances, Nano (free plan) and Micro (paid plans), with additional compute instance sizes available if you need extra performance when scaling up.
 
-<Admonition type="note" label="Upgrade to Micro Instances at No Additional Charge for Paid Projects">
+<Admonition type="note" label="Nanos in pro orgs">
 
-If you upgraded your Supabase organization from the Free plan to a paid plan, any projects created on the Free plan will remain on Nano instances due to the upgrade process [incurring downtime](/docs/guides/platform/compute-and-disk#upgrade-downtime).
-
-Since every project on a paid plan is already paying for a Micro instance you can upgrade to Micro at no additional charge. See [Supabase Pricing](https://supabase.com/pricing) for more information.
+In Pro organizations, Nano's are billed as Micros, so we recommend upgrading them when it's convenient for you. See [Supabase Pricing](https://supabase.com/pricing) for more information.
```

Comment:
```suggestion
In Pro organizations, Nano instances are billed at the same price as Micro instances, so we recommend upgrading your Nano instances when it's convenient for you. We don't auto-upgrade because of the downtime incurred. See [Supabase Pricing](https://supabase.com/pricing) for more information.
```